addons/api/package.json

```diff
@@ -23,18 +23,19 @@
     "doc:toc": "doctoc README.md"
   },
   "dependencies": {
+    "@ember/test-waiters": "^4.1.0",
     "dexie": "^3.2.4",
-    "ember-auto-import": "^2.8.1",
+    "ember-auto-import": "^2.10.0",
     "ember-can": "^4.2.0",
     "ember-cli-babel": "^8.2.0",
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-mirage": "^3.0.3",
-    "ember-data": "~4.12.0",
+    "ember-data": "~5.3.12",
     "miragejs": "^0.1.48",
     "tracked-built-ins": "^3.1.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2",
+    "@babel/core": "^7.26.10",
```

Comment:
This is not related to ember-data upgrade, just noticed it was already resolving to 7.26.10.